ui/js/keypad.js

```diff
@@ -92,7 +103,7 @@ Keypad.prototype.backspace = function backspace () {
 
   this.result = this.result.substring(0, this.result.length - 1)
 
-  var display = getDisplay(this.result, this.type, this.opts.country)
+  var display = getDisplay(this.result, this.type, this.opts.country, this.opts)
```

Comment:
Since you're passing `this.opts` as a parameter now, we can omit the `this.opts.country` and get it inside the `getDisplay()` function